source/docs/advanced/payments/adyen.md

```diff
@@ -5,46 +5,67 @@ title: Adyen integration
 
 This page contains information about the different ways you can accept payments with [Adyen](https://www.adyen.com/) in your Front-Commerce application.
 
-## Magento2 module
+## As a Front-Commerce payment (no Magento modules needed)
```

Comment:
Maybe have this title more related to the magneto section so that it's more apparent that they are different.
```diff
- ## As a Front-Commerce payment (no Magento modules needed)
+ ## As a Front-Commerce payment method
```

and then for the next section:
```diff
- ## Magento2 module
+ ## As a Magento2 Integration
```

It might be good to add a description at line 7 explaining the difference between `As a Front-Commerce payment` and `As a Magento2 Integration`, almost like a table of content?